Dockerfile.template

```diff
@@ -13,9 +13,13 @@ LABEL maintainer="Community & Partner Engineering Team <community-partner@circle
 ENV PG_VER=%%VERSION_FULL%%
 ENV PG_MAJOR=%%VERSION_MAJOR%%
 ENV POSTGRES_HOST_AUTH_METHOD=trust
+ENV PGTAP_VERSION v1.2.0
 
 USER root
 RUN apt-get update && apt-get install -y --no-install-recommends \
+		git \
```

Comment:
could you please remove git, make,and perl unless you have a reason to keep? these all technically already exist since these are being built off cimg-base